diff --git a/base/BUILD.gn b/base/BUILD.gn
index b7567e3d6b4dbeb86680931eddb10387a4ab827e..5dbc821f2f51c2d4eb0233ff868e03509d63c159 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4453,6 +4453,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/ThreadPoolTaskExecutor.java",
       "android/java/src/org/chromium/base/task/UiThreadTaskExecutor.java",
     ]
+    sources += brave_java_base_sources
 
     if (use_clang_profiling) {
       sources += [
@@ -4491,6 +4492,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_aidl("process_launcher_aidl") {
