diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 1ed8d55253bb39bf6748d8761c89d0fa2385047d..0680c3939c376dda563bc0702c19d976eb2dc90e 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3906,6 +3906,7 @@ const flags_ui::FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
