diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index d731be2776e02964d2dbc66e0e2b1775a9311d59..d0ff90ee49c306c11b4e87f02623fcdfe9404173 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -706,6 +706,7 @@ static_library("ui") {
 
     "//chrome/browser/permissions",
   ]
+  public_deps += [ "//brave/browser/ui" ] allow_circular_includes_from += [ "//brave/browser/ui" ]
 
   if (enable_vr && is_win) {
     deps += [ "//chrome/browser/vr:vr_base" ]
@@ -4569,10 +4570,13 @@ static_library("ui") {
       ]
       deps += [ "//chrome/updater/app/server/win:updater_legacy_idl_idl" ]
     } else {
-      sources += [
-        "webui/help/version_updater_basic.cc",
-        "webui/help/version_updater_basic.h",
-      ]
+      if (!is_official_build) {
+        # In official build, //brave/browser/ui handles.
+        sources += [
+          "webui/help/version_updater_basic.cc",
+          "webui/help/version_updater_basic.h",
+        ]
+      }
     }
   } else {  # 'OS!="win"
     if (toolkit_views) {
