diff --git a/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc b/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
index c1ae0743981e79c904e98dc35d9cdce70709962a..f2af0ee4980d9e4da435cf1d58e2199770d9fe81 100644
--- a/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
+++ b/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
@@ -661,6 +661,7 @@ SkPath IconLabelBubbleView::GetHighlightPath() const {
   const float corner_radius = highlight_bounds.height() / 2.f;
   const SkRect rect = RectToSkRect(highlight_bounds);
 
+  BRAVE_ICON_LABEL_BUBBLE_VIEW_GET_HIGHLIGHT_PATH
   return SkPath().addRoundRect(rect, corner_radius, corner_radius);
 }
 
