diff --git a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java
index f448ee3de15cb1875d9f2552e89b39cfe9a3e858..ee1f151231211fa334d132d15b97d78ec429490b 100644
--- a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java
+++ b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarPhone.java
@@ -442,6 +442,7 @@ public class ToolbarPhone extends ToolbarLayout
         if (mOptionalButtonCoordinator != null) {
             mOptionalButtonCoordinator.setBackgroundColorFilter(color);
         }
+        BraveToolbarLayout.class.cast(this).updateModernLocationBarColorImpl(color);
     }
 
     private void updateModernLocationBarCorners() {
@@ -616,6 +617,7 @@ public class ToolbarPhone extends ToolbarLayout
                         .notifyEvent(EventConstants.PARTNER_HOME_PAGE_BUTTON_PRESSED);
             }
         }
+        BraveToolbarLayout.class.cast(this).onClickImpl(v);
     }
 
     @Override
@@ -1110,6 +1112,7 @@ public class ToolbarPhone extends ToolbarLayout
      * New Tab Page.
      */
     private void updateLocationBarLayoutForExpansionAnimation() {
+        if (!BraveToolbarLayout.class.cast(this).isLocationBarValid(mLocationBar)) return;
         TraceEvent.begin("ToolbarPhone.updateLocationBarLayoutForExpansionAnimation");
         if (isInTabSwitcherMode()) return;
 
@@ -1560,6 +1563,7 @@ public class ToolbarPhone extends ToolbarLayout
             canvas.restore();
         }
 
+        BraveToolbarLayout.class.cast(this).drawAnimationOverlay(mToolbarButtonsContainer, canvas);
         // Draw the tab stack button and associated text if necessary.
         if (mTabSwitcherAnimationTabStackDrawable != null
                 && mToggleTabStackButton != null
@@ -2402,6 +2406,7 @@ public class ToolbarPhone extends ToolbarLayout
         } else {
             populateUrlClearExpansionAnimatorSet(animators);
         }
+        if (!BraveToolbarLayout.class.cast(this).isLocationBarValid(mLocationBar)) return;
         mUrlFocusLayoutAnimator = new AnimatorSet();
         mUrlFocusLayoutAnimator.playTogether(animators);
 
