diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 52ecc40da02264e8f4b347bc5a131aa41446bbc5..d836fb61641745d893fe52f4fb12f96125ecc8a5 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3957,6 +3957,7 @@ const flags_ui::FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
