diff --git a/chrome/browser/extensions/extension_context_menu_model.cc b/chrome/browser/extensions/extension_context_menu_model.cc
index d5e3f14b43b440188dbf483853883aa69274e149..db8038b727932eef520b5d6289b57d61fe3d0574 100644
--- a/chrome/browser/extensions/extension_context_menu_model.cc
+++ b/chrome/browser/extensions/extension_context_menu_model.cc
@@ -813,7 +813,7 @@ void ExtensionContextMenuModel::InitMenu(const Extension* extension,
   }
 
   const ActionInfo* action_info = ActionInfo::GetExtensionActionInfo(extension);
-  if (delegate_ && !is_component_ && action_info && !action_info->synthesized &&
+  if (delegate_ && action_info && !action_info->synthesized &&
       profile_->GetPrefs()->GetBoolean(prefs::kExtensionsUIDeveloperMode)) {
     AddSeparator(ui::NORMAL_SEPARATOR);
     AddItemWithStringId(INSPECT_POPUP, IDS_EXTENSION_ACTION_INSPECT_POPUP);
