diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 0fb94c9b9b67e3b6d1e6e2aa1cfb31765a8d9ccd..dd198f09dc0680140dee8ba50c1dc708d26ca9cf 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2656,6 +2656,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
