diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index e00f2321b7ae9797e336cf7afbbce79a5a5e2b4b..29cf66b98cc22b6421c5b97b49df93aba56524b7 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2657,6 +2657,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
