diff --git a/components/permissions/request_type.cc b/components/permissions/request_type.cc
index 1bfd3e817095bb218c0ad0200ac4cc0191aebbde..740d35b02f3b0f729ba24dd4011e60ca1fefb10c 100644
--- a/components/permissions/request_type.cc
+++ b/components/permissions/request_type.cc
@@ -433,6 +433,7 @@ const char* PermissionKeyForRequestType(permissions::RequestType request_type) {
         return "window_management";
       }
 #endif
+    BRAVE_PERMISSION_KEY_FOR_REQUEST_TYPE
   }
 
   return nullptr;
