diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index d776a0e4ec62fce018e0f930bca45d0303d8b11b..de58837eedd0392a9e0ae7f72311f9e8fd9bda5a 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -861,6 +861,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
