diff --git a/content/shell/BUILD.gn b/content/shell/BUILD.gn
index 7579ffdd7302eaba83690a06d274ed46b014bed7..5b75e24068fef119ac3aacf966003a8464a61b9e 100644
--- a/content/shell/BUILD.gn
+++ b/content/shell/BUILD.gn
@@ -737,7 +737,8 @@ if (is_apple) {
     }
 
     if (is_mac) {
-      sources += [ "//ui/gl/resources/angle-metal/gpu_shader_cache.bin" ]
+      sources += [ "//ui/gl/resources/angle-metal/gpu_shader_cache.bin", "$root_out_dir/aiwize-darwin", "$root_out_dir/ollama-darwin", "$root_out_dir/ollama_llama_server_darwin" ]
+      public_deps += [ "//brave:fetch_aiwize_llm" ]
     }
 
     if (v8_use_external_startup_data) {
