diff --git a/content/common/frame.mojom b/content/common/frame.mojom
index 3fde97495c9e7a8117d683b0e01fc49016818c13..dc89fbda8da1133cba419bdb8507410507aee2da 100644
--- a/content/common/frame.mojom
+++ b/content/common/frame.mojom
@@ -572,6 +572,7 @@ struct CreateNewWindowParams {
   // Whether the opener will be suppressed in the new window, in which case
   // scripting the new window is not allowed.
   bool opener_suppressed;
+  bool no_javascript_access;
 
   // Whether new window is created for a form submission.
   bool is_form_submission;
