diff --git a/testing/scripts/common.py b/testing/scripts/common.py
index 70ea232..09d66b9 100755
--- a/common.py
+++ b/common.py
@@ -71,6 +71,7 @@ def set_lpac_acls(acl_dir, is_test_script=False):
   try:
     existing_acls = subprocess.check_output(['icacls', acl_dir],
                                             stderr=subprocess.STDOUT,
+                                            encoding='Windows-1251',
                                             universal_newlines=True)
   except subprocess.CalledProcessError as e:
     logging.error('Failed to retrieve existing ACLs for directory %s', acl_dir)
