diff --git a/third_party/blink/common/permissions/permission_utils.cc b/third_party/blink/common/permissions/permission_utils.cc
index a5e49bb81d511489b869a2554c965b56fdd9da84..ce025a39fd36e114da352c85efd121e713d62644 100644
--- a/third_party/blink/common/permissions/permission_utils.cc
+++ b/third_party/blink/common/permissions/permission_utils.cc
@@ -28,6 +28,7 @@ mojom::PermissionStatus ToPermissionStatus(const std::string& status) {
 
 std::string GetPermissionString(PermissionType permission) {
   switch (permission) {
+    PERMISSION_UTIL_GET_PERMISSION_STRING
     case PermissionType::GEOLOCATION:
       return "Geolocation";
     case PermissionType::NOTIFICATIONS:
